src/main/java/com/google/api/codegen/LanguageUtil.java

```diff
@@ -67,4 +67,8 @@ public static String lowerUnderscoreToLowerCamel(String name) {
   public static String lowerCamelToUpperCamel(String name) {
     return CaseFormat.LOWER_CAMEL.to(CaseFormat.UPPER_CAMEL, name);
   }
+
+  public static String lowerCamelToLowerUnderscore(String name) {
```

Comment:
It's probably out of the scope of this CL, but can we consolidate these two implementations and keep it only in CodegenContext?
LanguageUtil functions are only used by GoGapicContext.java and JavaContextCommon.java, and it's quite easy to rewrite GoGapicContext -- although I'm a bit unsure how JavaContextCommon can be fixed.